spark3/delta-streaming-dupes.dedup-cdf.scala

```diff
@@ -0,0 +1,156 @@
+// Spark: 3.5.1
+// Local: --driver-memory 1G --master 'local[2]' --packages io.delta:delta-spark_2.12:3.1.0 --conf spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension --conf spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog
+
+// COMMAND ----------
+
+/*
+
+This snippet presents a use case of streaming over a delta table where we do merges that updates 1 record.
```

Comment:
Could you please explain in this section how it may represent a performance issue?